### ISSUE TITLE
Enable Go metrics in service-catalog controller-manager 

### ISSUE THREAD

**Issue by piotrmiskiewicz** [kyma-project/kyma#1043]
AC:
Grafana dashboard with Go metrics for service-catalog controller-manager.

Hint:
see 'controllerManager.enablePrometheusScrape' value

**Comment by mszostok** [comment]
Please see [this PR]( https://github.com/kyma-project/kyma/pull/1386) to check introduced modifications.

TL;DR;
The Service Catalog Manager expose only the business metrics (they are always available - hardcoded in source code). The are three types of metrics:
- BrokerServiceClassCount - Number of services classes by Broker.
- BrokerServicePlanCount  -  Number of services classes by Broker.
- OSBRequestCount - Cumulative number of HTTP requests from the OSB Client to the specified Service Broker grouped by broker name, broker method, and response status.

source: https://github.com/kubernetes-incubator/service-catalog/blob/v0.1.34/pkg/metrics/metrics.go#L77-L83

this functionality was introduced by this PR: https://github.com/kubernetes-incubator/service-catalog/pull/1608/files

The **controllerManager.enablePrometheusScrape** flag in **values.yaml** file from ServiceCatalog Helm Chart refers only to the Prometheus configuration. Setting it to `true` will direct Prometheus to automatically scrape custom metrics exposed from Service Catalog because it's looking for pods with the prometheus.io/scrape annotation. But we are using the Prometheus operator and 
> The annotation is not supported by the operator. ServiceMonitors should be
used instead. The annotation is very limited by its nature and there's no
feasible way to support anything beyond "scrape on" and a single port. 

source: https://github.com/coreos/kube-prometheus/pull/16#issuecomment-305933103

because of that, we are using the ServiceMonitor. 

In this task I've added only new graphs for the business metrics for Controller manager. If we want to have also the Go metrics or add more business metrics then we need to contribute to the ServiceCatalog.
